breol_app_warning/js/dialog.js

```diff
@@ -0,0 +1,9 @@
+; jQuery(function () {
+  if (Drupal.settings.breol_app_warning) {
```

Comment:
Should this have been `if (Drupal.settings. hasOwnProperty('breol_app_warning'))` just to be sure to be sure.

Reply:
This is the way it's usually done throughout Drupal (and JavaScript in general). We could use `hasOwnProperty`, but then we'd need an additional check to see if it's also truthy.

Reply:
But would JS not stop execution if the property is not there?

Reply:
Okay, testet it and it works... but it's not pretty.